how-to/register-with-browser/public/dos.json

```diff
@@ -0,0 +1,14 @@
+{
+  "desktopSettings": {
+    "systemApps": {
+      "workspace": {
+        "version": "5.5.0",
```

Comment:
Right now the main branch is aligned with version 6.1.6

Reply:
updated to 6.1.6